<?xml version="1.0"?>
<launch>

  <node pkg="tf" type="static_transform_publisher" name="world_to_cam" 
    args="0 0 0.5 -1.57 0 -1.57 world usb_cam 10" />
  <node ns="camera" pkg="image_proc" type="image_proc" name="image_proc"/>
  <node name="ar_pose_front" pkg="ar_pose" type="ar_single" respawn="false"
    output="screen">
   <param name="marker_frame" type="string" value="marker_2"/>
    <param name="marker_pattern" type="string" value="data/4x4/4x4_95.patt"/>
    <param name="marker_width" type="double" value="150"/>
    <param name="marker_center_x" type="double" value="0.0"/>
    <param name="marker_center_y" type="double" value="0.0"/>
    <param name="threshold" type="int" value="100"/>
    <param name="use_history" type="bool" value="true"/>
   <param name="reverse_transform" type="bool" value="false"/>
   <remap from="/camera/camera_info" to="/ardrone/front/camera_info"/>
    <remap from="/camera/image_raw" to="/ardrone/front/image_raw"/>
    <remap from="/ar_pose_marker" to="/ar_pose_marker_front"/>
    
  </node>
</launch>
